app/Models/Aircraft.php

```diff
@@ -137,34 +141,34 @@ public function landingTime(): Attribute
     }
 
     /**
-     * foreign keys
+     * Relationships
      */
-    public function airline()
+    public function airline(): ZnckBelongsToThrough
```

Comment:
What's this `ZnckBelongsToThrough`?

Reply:
It is Staudenmeir's BelongsToThrough relationship, which we added and using many months ago. 

https://github.com/staudenmeir/belongs-to-through

I had to give a name to the return check cause we are already using `BelongsToThrough` naming as the `Trait`

Reply:
Ok cool, I didn't see any changes to the composer file so that's why I was wondering. LGTM